doc/060_forget.rst

```diff
@@ -392,3 +392,30 @@ The ``prune`` command accepts the following options:
 -  ``--dry-run`` only show what ``prune`` would do.
 
 -  ``--verbose`` increased verbosity shows additional statistics for ``prune``.
+
+
+Recovering from "no free space" errors
+**************************************
+
+In some cases when a repository has grown large enough to fill up all disk space or the
+allocated quota, then ``prune`` might fail to free space. ``prune`` works in such a way
+that a repository remains usable no matter at which point the command is interrupted.
+However, this also means that ``prune`` requires some scratch space to work.
+
+In most cases it is sufficient to instruct ``prune`` to use as little scratch space as
+possible by running it as ``prune --max-repack-size 1``. Obviously, this can only work
```

Comment:
Here too, shouldn't that say `--max-repack-size 0`? And maybe add the hint that for versions earlier than 0.13.0 `--max-repack-size 1` should be used.

Reply:
I've applied both suggestions.